railties/lib/rails/command.rb

```diff
@@ -0,0 +1,96 @@
+require "active_support"
+require "active_support/core_ext/enumerable"
+require "active_support/core_ext/object/blank"
+require "active_support/core_ext/hash/transform_values"
+
+require "thor"
+
+module Rails
+  module Command
+    autoload :Behavior, "rails/command/behavior"
+    autoload :Base, "rails/command/base"
```

Comment:
Because those ones use ActiveSupport::Autoload (on my phone, from memory). This is the standard in the code base, conventional file layout and no paths.